.circleci/config.yml

```diff
@@ -1454,46 +1454,6 @@ workflows:
               ignore:
                 - nightly
                 - postnightly
-      - mac_test:
-          name: macos-12-py3-x86-64-test-1-2-default
```

Comment:
dunno if circleci lets you do this, but `if: false` style syntax is nice for disabling workflows while keeping it in the code